src/standard/utils.html

```diff
@@ -306,6 +306,18 @@
         }
       }
       return elt;
+    },
+
+    /**
+     * Convenience method for checking whether an element is in this element's
```

Comment:
For consistency, we should start with a verb and end with a period, like:

    Checks whether an element is in this element's shadow DOM tree.

Or:

    Checks whether _node_ is in this element's shadow DOM tree.